beancount_import/source/schwab_csv.py

```diff
@@ -1113,10 +1295,17 @@ def _load_brokerage_transactions(reader: csv.DictReader, account: str,
                 price=price,
                 fees=fees,
                 amount=Amount(amount, currency=CASH_CURRENCY) if amount else None,
+                merger_symbol=merger_symbol,
+                merger_quantity=merger_quantity,
+                merger_description=merger_desc,
                 filename=filename,
                 line=lno + 2,
             )
         )
+        if merger_symbol is not None:
+            merger_symbol = None
+            merger_quantity = None
+            merger_desc = None
```

Comment:
If you wanted to avoid this one idea is to modify the loop like I do here: https://github.com/bayesianmind/beancount-import/blob/master/beancount_import/source/schwab_equityaward.py#L108

I think the way you've done it here is the least bad option FWIW.

Reply:
I agree, this doesn't seem bad at all to me. I think it makes sense in this case to do this line merging here in the CSV parsing, even though it's a slight departure from the "RawEntries are 1:1 representations of rows in the CSV" design.

One small tweak I might make is to store the three items as a tuple or NamedTuple, so that you can just have a single variable to set to either None or that tuple, and then not have to arbitrarily choose one of them to check for None-ness.

Reply:
Introduced the nametuple.